dev/storage_traits.h

```diff
@@ -42,227 +39,83 @@ namespace sqlite_orm {
             template<class S, class O>
             struct storage_columns_count : storage_columns_count_impl<storage_find_impl_t<S, O>> {};
```

Comment:
@fnc12 `storage_columns_count`, `storage_foreign_keys_count`, `storage_fk_references`, `storage_foreign_keys` are not used, only in unit tests. Should we move them to unit tests or even remove them completely?

Reply:
let's move them to ut. We must not have code which is never used but it can be helpful so let's leave it in ut. If it is not required during next two versions then let's remove it

Reply:
Do you want to keep the `storage_traits` nested namespace?

Reply:
yes